torch/special/__init__.py

```diff
@@ -820,3 +851,17 @@
     tensor([1., 1., 1.])
 
 """.format(**common_args))
+
+chebyshev_polynomial_u = _add_docstr(_special.special_chebyshev_polynomial_u,
+                                     r"""
+chebyshev_polynomial_t(input, n, *, out=None) -> Tensor
+
+Chebyshev polynomial of the second kind :math:`U_{n}(\text{input})`.
```

Comment:
It'd be nice to elaborate here -- we can use LaTex -- doesn't have to be in this PR